src/modules/AuthModule/components/Register/Register.tsx

```diff
@@ -150,34 +149,31 @@ const appendToFormData=(data:FormValues)=>{
         )}
         </Stack>
          <Stack my="5px" mr="5px" >
-         <label htmlFor="country"  style={{ fontSize: "1.2rem", fontWeight: 300, opacity: 0.8 }}>Country</label>
+         <InputLabel htmlFor="country" style={{ color:'rgba(21, 44, 91, 1)',fontSize: "1.1rem", fontWeight: 400, marginBottom: 6}}>Country</InputLabel>

```

Comment:
The TextField contains a Label Prop for the label. If we want to make our own TextField, we will use InputLabel, Input component (has multiple variations OutlinedInput ... ), and the HelperText components. But here we are using InputLabel with the TextField which alread has  label.